Makefile

```diff
@@ -235,19 +228,30 @@ CLEAN_TARGETS += clean-vendor
 clean-vendor:
 	-rm -rf $(VENDOR_DIR)
 
-CLEAN_TARGETS += clean-glide-cache
-.PHONY: clean-glide-cache
-## Removes the ./glide directory.
-clean-glide-cache:
-	-rm -rf ./.glide
-
-$(VENDOR_DIR): glide.lock glide.yaml
-	$(GLIDE_BIN) --quiet install --strip-vendor
-	touch $(VENDOR_DIR)
-
 .PHONY: deps
 ## Download build dependencies.
-deps: $(VENDOR_DIR)
+deps: $(DEP_BIN) $(VENDOR_DIR)
+
+# install dep 
+$(DEP_BIN):
```

Comment:
I don't get how you install dep into `$(DEP_BIN_DIR)` here. I think it is download into the current directory and NOT: to `$(DEP_BIN_DIR)`

1. You create the `$(DEP_BIN_DIR)`.
2. Download dep in current dir.
3. You switch current directory to `$(DEP_BIN_DIR)`.
4. You do checksum checks.
5. You assign proper execution rights to the.